Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -142,62 +142,62 @@ void AdvancedWidget::OnEmulationStateChanged(bool running)
 void AdvancedWidget::AddDescriptions()
 {
   static const char TR_WIREFRAME_DESCRIPTION[] =
-      QT_TR_NOOP("Render the scene as a wireframe.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Renders the scene as a wireframe.\n\nIf unsure, leave unchecked.");
   static const char TR_SHOW_STATS_DESCRIPTION[] =
-      QT_TR_NOOP("Show various rendering statistics.\n\nIf unsure, leave this unchecked.");
-  static const char TR_TEXTURE_FORMAT_DECRIPTION[] =
-      QT_TR_NOOP("Modify textures to show the format they're encoded in. Needs an emulation reset "
-                 "in most cases.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Shows various rendering statistics.\n\nIf unsure, leave unchecked.");
+  static const char TR_TEXTURE_FORMAT_DECRIPTION[] = QT_TR_NOOP(
+      "Modifies textures to show the format that they are encoded in.\n\nNeeds "
+      "an emulation reset in most cases.\n\nIf unsure, leave unchecked.");
```

Comment:
Does this actually need an emulation reset? Can someone verify this? I've toggled the option after game start with success before.

Reply:
We could reword it to "May require an emulation reset to apply"

Reply:
I'm happy with that.